From 58114dab90be08e28bb97ba767e29b44636e039b Mon Sep 17 00:00:00 2001
From: Roman Birg <roman@cyngn.com>
Date: Wed, 17 Aug 2016 11:20:56 -0700
Subject: [PATCH 14/16] Messaging: fix bad recycle on sending two mms in a row

When we are handling the add animation ourself, we need to call the
dispatchAdd() to finish the operation so the recycler view can propely
handle it.

Ticket: CYNGNOS-3253

Change-Id: Ia8755a1b10ac678c304ac7e1208336e11fbe7b77
Signed-off-by: Roman Birg <roman@cyngn.com>
---
 .../android/messaging/ui/conversation/ConversationFragment.java  | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/com/android/messaging/ui/conversation/ConversationFragment.java b/src/com/android/messaging/ui/conversation/ConversationFragment.java
index 419ad5d..2631118 100644
--- a/src/com/android/messaging/ui/conversation/ConversationFragment.java
+++ b/src/com/android/messaging/ui/conversation/ConversationFragment.java
@@ -565,6 +565,7 @@ public class ConversationFragment extends Fragment implements ConversationDataLi
                             @Override
                             public void run() {
                                 view.setAlpha(1);
+                                dispatchAddFinished(holder);
                             }
                         });
                     mPopupTransitionAnimation.startAfterLayoutComplete();
-- 
2.17.1

